src/components/Modals/PatientEditModal/PatientEditModal.tsx

```diff
@@ -1,24 +1,28 @@
-import React, { useState } from 'react'
+import React, { Dispatch, SetStateAction, useState } from 'react'
 import Button from 'react-bootstrap/esm/Button'
 import Form from 'react-bootstrap/esm/Form'
 import Modal from 'react-bootstrap/esm/Modal'
 import { useMsal } from '@azure/msal-react'
-import { updatePatient, useAuthRequest } from '../../../utils/api/calls'
+import { PatientProps, updatePatient, useAuthRequest } from '../../../utils/api/calls'
 
-function EditPatientModal({ patient, updatePatientList, show, hide }) {
+type Props = {
+  patientForm: PatientProps
+  setPatientForm: Dispatch<SetStateAction<PatientProps>>
+  updatePatientList: (patient: PatientProps) => void
+  show: boolean
+  hide: () => void
+}
+
+function PatientEditModal({ patientForm, setPatientForm, updatePatientList, show, hide }: Props) {
   const { instance } = useMsal()
   const request = useAuthRequest()
-  const [error, setError] = useState(false)
 
-  const [patientForm, setPatientForm] = useState({
-    firstName: patient?.firstName,
-    lastName: patient?.lastName,
-    birthdate: patient?.birthdate,
-  })
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Not sure if this is a comment but if it should be removed?

Reply:
It disables lint errors from this line. 
`  const [error, setError] = useState(false)`
Error is currently unused. We do set the error from API calls, but do not show them anywhere. As a result the 'error' variable in unused, but can be applied in the future.